src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -554,6 +554,10 @@
   <data name="MemberMatchNoMatchFound" xml:space="preserve">
     <value>No match found.</value>
   </data>
+  <data name="UnknownResourceType" xml:space="preserve">
```

Comment:
>UnknownResourceType [](http://example.com/codeflow?start=14&length=19)

there is `ResourceNotSupported`

Reply:
Yep, that looks good, better to not add a new one unnecesarily.